docs/contributing/github_integration.rst

```diff
@@ -0,0 +1,115 @@
+GitHub Integration
+====================
+
+buildtest has several github integration, including automated checks during PR that maintainers will check to
```

Comment:
nit: github integration**s**

Reply:
"including automated checks during PR that maintainers will check during the PR remove" (delete the "to" after check)